CHANGELOG.md

```diff
@@ -4,11 +4,53 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [1.0.1]
+## [1.2.0] - 2021-07-19
+
+:warning: This version drops support for PHP 5.2!
+
+### Changed
+* PHP 5.2 is no longer supported. The minimum supported PHP version for the WHIP library is now PHP 5.3. [#96]
+* The previous solution to prevent duplicate messages as included in v1.0.2 has been improved upon and made more stable. Props [Drew Jaynes]. [#44]
+* The `Whip_InvalidOperatorType::__construct()` method now has a second, optional `$validOperators` parameter. [#62]
+    If this parameter is not passed, the default set of valid operators, as was used before, will be used.
+* Improved protection against XSS in localizable texts. [#50]
+* Improved support for translating localizable texts (I18n). [#59]
+* The distributed package will no longer contain development related files. [#45]
```

Comment:
```suggestion
* The distributed package will no longer contain development-related files. [#45]
```

Reply:
Happy to change it, but the dashes are a "dutchism" and not expected in English.

Reply:
Grammarly proposes the hyphen as well. Also see the section about `Compound adjectives` here: https://www.lexico.com/grammar/hyphen

Reply:
Fair enough. In that case: done. 😁